src/component/handlers/edit/setClipboardData.js

```diff
@@ -34,6 +35,11 @@ function setClipboardData(e: SyntheticClipboardEvent, editor: DraftEditor, {text
       doCopyTrap(editor, html);
       // Let the native event go through
       return;
+    } else {
+      if (needsClipboardPolyfill()) {
```

Comment:
nit: why not just add this check to the above condition since both result in the same code being run?

Reply:
:mind_blown: Good call!